src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Components/DemoPage.razor.scss

```
@@ -19,15 +19,20 @@
         align-items: center;
         margin-bottom: $doc-space-md;
         padding: $doc-space-2xs $doc-space-sm;
-        top: calc(var(--demo-header-height) + 0.35rem + #{$bit-env-inset-top});
+        // Both halves of the bar's box come from app.scss rather than from the toggle inside it: the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)